test/gurka/test_route.cc

```diff
@@ -218,12 +218,13 @@ TEST(AutoDataFix, deprecation) {
   ParseApi(request_str, Options::route, request);
 
   EXPECT_EQ(request.options().costing(), valhalla::auto_);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).ignore_access(), true);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).ignore_closures(), true);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).ignore_oneways(), true);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).ignore_restrictions(), true);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).use_ferry(), 0.1f);
-  EXPECT_EQ(request.options().costing_options(valhalla::auto_).use_tolls(), 0.77f);
+  const auto& co = request.options().costing_options().find(valhalla::auto_)->second;
```

Comment:
lots of these kinds of changes as well. it was easier to get the costing options once and then test the individual fields